src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -135,6 +135,9 @@ export default function VideoVisitLocation({ header, appointment, facility }) {
           className="vads-u-display--block"
           backgroundOnly
         >
+          <h2 className="vads-u-font-size--h3 vads-u-font-size--base vads-u-margin-top--0">
```

Comment:
Same question as above

Reply:
Nope. Just cut and paste from some other code. I will change to use props. The default level is 3 with is fine.

Reply:
I think 2 was the intent there, but I could be wrong.